test/cases/image_tests.sh

```diff
@@ -5,6 +5,12 @@ set -euo pipefail
 source /etc/os-release
 ARCH=$(uname -m)
 DISTRO_CODE="${DISTRO_CODE:-${ID}_${VERSION_ID//./}}"
+BUILD_ID="${BUILD_ID:-${CI_PIPELINE_ID}}"
+
+# Set DISTRO_CODE for rhel-8 runners
+if [[ "$CI_JOB_NAME" =~ .*rhel-8-.* ]]; then
+    DISTRO_CODE="rhel_8"
+fi
```

Comment:
I don't understand why we need this change ?

Reply:
When this test was ran in Jenkins it had `DISTRO_CODE` specified for each stage. Without this change, when it's ran on regular rhel-8 runner, which is currently 8.3 it will fail with `ls: cannot access 'rhel_83-x86_64*.json': No such file or directory` because there is no 8.3 manifest... What this does is if the runner is regular RHEL 8, it sets the `DISTRO_CODE` to `rhel_8` ... This is what I came up with, it should work when we update the runners to use RHEL 8.4 because they'll be still be called rhel 8 runners. So no hard coding for 8.3 and then having to change when we update...

Reply:
This change and not having `BUILD_ID` or `CI_BUILD_ID` defined, e.g. if I simply just install `osbuild-composer-tests` from the S3 repo, makes is impossible to simply execute `image-tests.sh` locally

Reply:
The same goes for `CI_JOB_NAME`. I would suggest to keep this test cases ideally working out of the box even outside of the CI environment...

Reply:
Ok so I removed that change, I guess it's better to define `DISTRO_CODE` in .gitlab-ci.yml file if it needs to be something different. Also I don't think this worked outside CI before as `BUILD_ID` was just set to `BUILD_ID` and I believe that comes from Jenkins. I changed it to some random uuid though in case it's not defined so it should work outside CI now.